web/src/components/Editor/Editor.tsx

```diff
@@ -46,16 +47,24 @@ const Editor = forwardRef((props: Props, ref: React.ForwardedRef<EditorRefAction
       focus: () => {
         editorRef.current?.focus();
       },
-      insertText: (rawText: string) => {
+      insertText: (content = "", prefix = "", suffix = prefix) => {
```

Comment:
```suggestion
      insertText: (content = "", prefix = "", suffix = prefix) => {
```
The third param is equal the second defaultly, So it can be ignore

Reply:
sorry for do not see that